.github/workflows/build.yml

```diff
@@ -185,7 +197,7 @@ jobs:
       run: make buildbottest TESTOPTS="-j4 -uall,-cpu"
 
   build_ubuntu:
-    name: 'Ubuntu'
+    name: 'Build on Ubuntu'
```

Comment:
This change causes this required check to be never met:

<img width="843" alt="image" src="https://user-images.githubusercontent.com/1324225/235838297-7f42355f-9e14-4fe6-8927-d91ad47bd1eb.png">


Let's keep the original name, otherwise we'll need to sync all the other open PRs:

```suggestion
    name: 'Ubuntu'
```

Reply:
Ah, this was actually an accidental artifact from my original approach, which was to do the build on Ubuntu once, upload it as an artifact, then download it in two separate jobs, one that ran the test suite with hypothesis and one without.

I didn't immediately get it to work and it didn't seem on the critical path so I reverted it, but I guess the name got changed in the shuffle, I've changed it back now.